lib/icon-button/index.tsx

```diff
@@ -10,15 +10,22 @@ type OwnProps = {
 type Props = OwnProps;
 
 export const IconButton = ({ icon, title, ...props }: Props) => (
-  <Tooltip
-    classes={{ tooltip: 'icon-button__tooltip' }}
-    enterDelay={200}
-    title={title}
-  >
-    <button className="icon-button" type="button" data-title={title} {...props}>
-      {icon}
-    </button>
-  </Tooltip>
+  <span>
```

Comment:
This fixes a console error when it's passed a disabled element. See https://material-ui.com/components/tooltips/#disabled-elements